src/pages/DataAccessRequestApplication_new.js

```diff
@@ -1062,7 +1094,12 @@ class DataAccessRequestApplicationNew extends Component {
               })
             ]),
 
-            div({ isRendered: this.state.step === 4 }, [
+            div({id: 'step-4', className: 'step-container'}, [
```

Comment:
You don't have to worry about the Progress Report component right now. The mocks have it set up, but we won't be implementing the feature on the back-end for a while due to Data Submitter and the DAR Form UI redesign being a higher priority.